translations/binaries/slps_strings_title.json

```
@@ -677,21 +677,21 @@
             "string": "小さな体をめいっぱい使って、[LINE]\n勇猛果敢に敵を撃つ。[LINE]\n小弓使いの女の子。[END]",
             "original_bytes_length": 69,
             "available_bytes_size": 71,
-            "translation": "작은 몸을 최대한 사용하여,[LINE]\n용감하게 적을 쏜다.[LINE]\n작은 활잡이 소녀.[END]"
+            "translation": "작은 몸을 힘껏 써서[LINE]\n용감무쌍하게 적을 쏜다.[LINE]\n작은 활잡이 소녀.[END]"
         },
         {
             "offset": 1822264,
             "string": "小弓ガール[END]",
             "original_bytes_length": 11,
             "available_bytes_size": 15,
-            "translation": "작은 활 소녀[END]"
+            "translation": "단궁소녀[END]"
```

```suggestion
            "translation": "단궁 소녀[END]"
```

약간 사자성어 느낌을 내는 게 의도였을까요? (띄어쓰기가 없길래 :eyes:)

---

원래 그런 의도가 맞았는데 맞춤법이라던가 이것저것 고려했을 때 띄어쓰기 해두는 편이 나을 것 같아서 적용했어요